### PR TITLE
examples: add etcd backup operator example yaml files

### PR DESCRIPTION
**Pull request by fanminshi**
[skip ci]

### CODE REVIEW

**Comment by hongchaodeng** [comment]
This shouldn't be merged until its features are implemented.

**Comment by fanminshi** [comment]
depends on https://github.com/coreos/etcd-operator/pull/1460

File: example/etcd-backup-operator/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: etcd-backup-operator
+    spec:
+      containers:
+      - name: etcd-backup-operator
+        image: gcr.io/coreos-k8s-scale-testing/etcd-backup-operator:fanmin
```
Comment (hongchaodeng) [inline example/etcd-backup-operator/deploy.yaml]: add `TODO: replace temporary image with official quay image`.

**Comment by hongchaodeng** [comment]
lgtm after nit

**Pull request merged**